common/dynasties/112000_quilboar.txt

```diff
@@ -16,6 +16,7 @@ quilboar6 = { name = "dynn_Blackspine" culture = quilboar }
 quilboar7 = { name = "dynn_Spearhide" culture = quilboar }
 quilboar8 = { name = "dynn_Tortusk" culture = quilboar }
 quilboar9 = { name = "dynn_Ramtusk" culture = quilboar }
-# 112018 = { name = "dynn_Thorncurse" culture = quilboar }
+thorncurse = { name = "dynn_Thorncurse" culture = quilboar }
```

Comment:
comment it out because `error.log`
> Dynasty 'dynn_Thorncurse-thorncurse' had no founder for coat of arms generation, it should have a pre-scripted coat of arms instead

It complains like this for historical dynasty without any character with it in `history/characters`